ursaspace/ursa-arcgis-toolbox
# VCM-7568 Enabling polygon tool for AOI selection

**What's Changed**

- Enabled polygon selection in the Analytic Ordering Toolbox
-- Tool tips for Area Of Interest and Buffer (km) have been updated
--- AOI Tool-tip now reads: "Point or Polygon Geometries Only"
--- Buffer (km) Tool-tip now reads: "The required image coverage area radius around the AOI point provided, in kilometers (km). Value must be between 1 and 5. The value is not used on polygon AOIs.
- Enabled polygon selection in the Tasking Toolbox
-- Tool tips for Area Of Interest and Buffer (km) have been updated
- Added new unit test to ensure the geojson helper properly processed a polygon shape file
- Updated .gitignore to ignore Visual Studio metadata directory
---
**To Test**

- Run all unit test (see readme)
- Submit an analytic order using a polygon AOI

Steps to Submit an analytic order using a polygon AOI:

1. In the Analytics toolbox select the "Create Feature" button to the right of the Area of Interest field (the pencil icon) and select "Polygons" (see first image below)
2. Add the polygon to the Map layer (seem second image below) this will add a "Analytics Area of Interest (Polygons) feature layer to the Contents pane (see third image below)
3. In the Analytics toolbox select the "Analytics Area of Interest (Polygons)" feature layer in the AOI dropdown
4. Complete the rest of the Analytics order form and hit run

Similar steps can be taken to submit a tasking order.

![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/114867d8-f6ed-424e-9cb9-69a74278a8c8)
Image 1: Analytics Toolbox

![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/091bbec6-6cb3-4f53-80d6-36a0c567a3ea)
Image 2: Map Layer

![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/d1620fca-42d9-4016-b90e-d2b02047f4f4)
Image 3: Contents Pane

![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/f73eb526-7688-417e-8198-c30ddaf99093)
Image 4: AOI Tool-tip Text

![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/77fc0b64-4e34-46d6-b253-6258245aefd5)
Image 5: Buffer (km) Tool-tip Text
